gratipay/utils/team_history.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from datetime import datetime
+from decimal import Decimal as D
+
+
+def get_end_of_year_totals(db, team, year):
```

Comment:
Might be a good idea to be clear on what format we're expecting arguments in: Is `team` supposed to be a `Team` object, or just the team slug? (The latter is the correct answer :1st_place_medal: )

I'd suggest using a docstring to describe what the arguments should be, and try passing `Team` objects around as far as possible.